### PR TITLE
gl_preprocess: fix some logic bugs

### PR DESCRIPTION
**Pull request by bkoropoff**

### CODE REVIEW

File: prboom2/src/gl_preprocess.c [hunk 1]
```diff
@@ -922,7 +922,7 @@ static void gld_GetSubSectorVertices(void)
   {
     subsector_t* ssector = &subsectors[i];
 
-    if (gld_TriangulateSubsector(ssector))
+    if (!gld_TriangulateSubsector(ssector))
```
Comment (kraflab) [inline prboom2/src/gl_preprocess.c:925]: What happened here? Btw the comment on this method says it only applies to glnodes, do you have a test map for this?

Comment (bkoropoff) [reply prboom2/src/gl_preprocess.c:925]: This got inverted during the refactoring that created `gld_TriangulateSubsector`.  I just happened to notice it while looking at something else.  I don't have any test map specifically for this code path.

Comment (kraflab) [reply prboom2/src/gl_preprocess.c:925]: Originally:
```c
    if (!(sec->flags & SECTOR_IS_CLOSED) ||
        (cont != NULL && !(cont->flags & SECTOR_IS_CLOSED)) ||
        triangulate_subsectors)
```

```c
static dboolean gld_TriangulateSubsector(subsector_t *ssec)
{
  sector_t *container = ssec->sector->gl_pp;
  return !(ssec->sector->flags & SECTOR_IS_CLOSED) ||
         (container && !(container->flags & SECTOR_IS_CLOSED)) ||
         triangulate_subsectors;
}
```
Am I missing something?

Comment (bkoropoff) [reply prboom2/src/gl_preprocess.c:925]: Huh.  Well, `continue` clearly skips triangulating the current subsector, so I guess the original logic was also reversed.

Comment (kraflab) [reply prboom2/src/gl_preprocess.c:925]: Strange 😅 
But let's leave it as it was before then, unless an issue comes up.

**Comment by bkoropoff** [comment]
This will get subsumed by #239

**Pull request closed without merging**